To: vim_dev@googlegroups.com
Subject: Patch 7.3.092
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.092
Problem:    Resizing the window when exiting.
Solution:   Don't resize when exiting.
Files:	    src/term.c


*** ../vim-7.3.091/src/term.c	2010-12-30 12:30:26.000000000 +0100
--- src/term.c	2010-12-30 12:14:48.000000000 +0100
***************
*** 3053,3062 ****
      int		old_Rows = Rows;
      int		old_Columns = Columns;
  
!     (void)ui_get_shellsize();
!     check_shellsize();
!     if (old_Rows != Rows || old_Columns != Columns)
! 	shell_resized();
  }
  
  /*
--- 3053,3065 ----
      int		old_Rows = Rows;
      int		old_Columns = Columns;
  
!     if (!exiting)
!     {
! 	(void)ui_get_shellsize();
! 	check_shellsize();
! 	if (old_Rows != Rows || old_Columns != Columns)
! 	    shell_resized();
!     }
  }
  
  /*
*** ../vim-7.3.091/src/version.c	2010-12-30 12:30:26.000000000 +0100
--- src/version.c	2010-12-30 14:47:04.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     92,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
57. You begin to wonder how on earth your service provider is allowed to call
    200 hours per month "unlimited."

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
